### PR TITLE
docs: Add additional warning about Windows path length issue

### PR DESCRIPTION
**Pull request by thewtex**
Addresses #67

Suggested-by: Addison Elliott <addison.elliott@gmail.com>

### CODE REVIEW

File: docs/Quick_start_guide.rst [hunk 1]
```diff
@@ -12,6 +12,18 @@ To install the ITK Python package::
     $ python -m pip install --upgrade pip
     $ python -m pip install itk
 
+If a pre-compiled wheel package is not found for your Python distribution, then it will
+attempt to build from source.
+
+.. note::
```
Comment (addisonElliott) [inline docs/Quick_start_guide.rst]: Maybe I'm unfamiliar with the syntax but what does `.. note::` signify? For me, I typically do **Note:** <INSERT_TEXT_HERE> to draw attention to it.

File: docs/Build_ITK_Python_packages.rst [hunk 2]
```diff
@@ -85,7 +85,14 @@ In a PowerShell prompt::
 	    -a----         4/9/2017  11:14 PM       63274441 itk-4.11.0.dev20170407-cp35-cp35m-win_amd64.whl
 	    -a----        4/10/2017   2:08 AM       63257220 itk-4.11.0.dev20170407-cp36-cp36m-win_amd64.whl
 
-We need to work in a short directory to avoid path length limitations on Windows, so the repository is cloned into C:\IPP. Also, it is very important to disable antivirus checking on the C:\IPP directory. Otherwise, the build system conflicts with the antivirus when many files are created and deleted quickly, which can result in Access Denied errors. Windows 10 ships with an antivirus application, Windows Defender, that is enabled by default.
+We need to work in a short directory to avoid path length limitations on
+Windows, so the repository is cloned into C:\IPP.
+
```
Comment (addisonElliott) [inline docs/Build_ITK_Python_packages.rst]: Why did you put some of the sentences on new lines? Doesn't this automatically wrap?

File: docs/Quick_start_guide.rst (hunk 1 above)
Comment (jcfr) [reply docs/Quick_start_guide.rst]: @addisonElliott This is specific to restructed text syntax. See http://docutils.sourceforge.net/docs/ref/rst/directives.html#note

It allows to generate the documentation available here: http://itkpythonpackage.readthedocs.io/en/latest/Quick_start_guide.html

File: docs/Build_ITK_Python_packages.rst (hunk 2 above)
Comment (thewtex) [reply docs/Build_ITK_Python_packages.rst]: This is to improve formatting in old-school text editors and improve navigation in these editors.

**Comment by thewtex** [comment]
Thanks for the review @addisonElliott !

**Pull request merged**